test/e2e/functional_test.go

```diff
@@ -813,6 +813,14 @@ func (s *FunctionalSuite) TestDataTransformation() {
 		})
 }
 
+func (s *FunctionalSuite) TestHTTPOutputs() {
+	s.Given().
+		Workflow("@testdata/http-outputs.yaml").
+		When().
+		SubmitWorkflow().
+		WaitForWorkflow(fixtures.ToBeSucceeded)
```

Comment:
I think we should actually check that the outputs are present correctly? 

You can use `ExpectWorkflow` for this:

```go
	WaitForWorkflow(fixtures.ToBeSucceeded).
	ExpectWorkflow(func(t *testing.T, _ *metav1.ObjectMeta, status *wfv1.WorkflowStatus) {
                // look in `status` for outputs
	})
```

Reply:
Hi, I've add ExpectWorkflow but some check failed, I'm fixing it now.